crates/re_space_view_tensor/src/space_view_class.rs

```diff
@@ -167,16 +171,18 @@ impl SpaceViewClass for TensorSpaceView {
         &self,
         ctx: &ViewerContext<'_>,
         ui: &mut egui::Ui,
-        state: &mut Self::State,
+        state: &mut dyn SpaceViewState,
         _space_origin: &EntityPath,
         _space_view_id: SpaceViewId,
         _root_entity_properties: &mut EntityProperties,
-    ) {
+    ) -> Result<(), SpaceViewSystemExecutionError> {
+        let state = state.downcast_mut::<ViewTensorState>()?;
```

Comment:
Tensor space view appears to be broken. I believe that's because a "correct" state is never created. Missing `new_state()` override?  I suspect other space view classes might be affected too.

Reply:
great catch. Kinda proves the point of why the previous indirection was actually useful :/

Reply:
we have that problem in a lot of places acutally! I'll just remove the default impl for `new_state` to easily catch that

Reply:
good idea 👍🏻